README.md

```diff
@@ -81,38 +146,59 @@ will be processed to:
 }
 ```
 
-## Pseudo class and elements
+Psuedo-classes are also dealt with correctly, because it's easy to statically determine.
 
 ```css
 .foo {
-	--foo-color: #ff0000;
+	--foo-color: red;
 	color: var(--foo-color);
 }
 
 .foo:hover {
-	--foo-color: #00ff00;
+	--foo-color: green;
 }
 ```
 
-will be processed to:
+The above will be transformed to this:
 
 ```css
 .foo {
-	color: #ff0000;
+	color: red;
 }
 
 .foo:hover {
-	color: #00ff00;
+	color: green;
 }
 ```
 
+It also works very well with `calc()`.
```

Comment:
Combine it with [`postcss-calc`](https://github.com/postcss/postcss-calc), which reduces `calc()` expressions, and using CSS variables can all of a sudden can feel much like using existing preprocessors today.

Here's the CSS demo I had:

```css
:root {
	--page-margin: 1em;
}

/* Easy centering */
.wrapper {
	width: calc(100% - 2 * var(--page-margin));
	margin-left: var(--page-margin);
	margin-right: var(--page-margin);
}
```

The exact equivalent in SCSS would look like this:

```scss
$page-margin: 1em;

// Easy centering
.wrapper {
	width: calc(100% - 2 * #{$page-margin});
	margin-left: $page-margin;
	margin-right: $page-margin;
}
```

It's small, but insanely useful and one of those things those of us currently using preprocessors take for granted. And you never know what you have until it's gone (i.e. you are stuck with plain CSS when you're used to preprocessors).